src/flp/szk.rs

```diff
@@ -690,18 +690,16 @@ mod tests {
 
     fn generic_szk_test<T: Type>(typ: T, encoded_measurement: &[T::Field], valid: bool) {
         let mut nonce = [0u8; 16];
-        let mut verify_key = [0u8; 16];
+        let mut verify_key = [0u8; 32];
         let algorithm_id = 5;
         let szk_typ = Szk::new_turboshake128(typ.clone(), algorithm_id);
         thread_rng().fill(&mut verify_key[..]);
         thread_rng().fill(&mut nonce[..]);
-        let prove_rand_seed = Seed::<16>::generate().unwrap();
-        let helper_seed = Seed::<16>::generate().unwrap();
-        let leader_seed_opt = if szk_typ.requires_joint_rand() {
-            Some(Seed::<16>::generate().unwrap())
-        } else {
-            None
-        };
+        let prove_rand_seed = Seed::generate().unwrap();
+        let helper_seed = Seed::generate().unwrap();
+        let leader_seed_opt = szk_typ
+            .requires_joint_rand()
+            .then_some(Seed::generate().unwrap());
```

Comment:
I feel like Clippy is going to add a lint for this in the future.
```suggestion
            .then(|| Seed::generate().unwrap());
```